src/App.js

```diff
@@ -186,19 +236,34 @@ class App extends Component {
         <BackTop />
         <Content>
           <div style={{ background: '#fff', padding: 24, minHeight: 280 }}>
+
+          <div style={{ textAlign: 'right', paddingTop: 16, paddingBottom: 16 }}>
+            <Switch checkedChildren="Simple" unCheckedChildren="Detailed" />
+          </div>
+
           <Search
             placeholder="please enter link here"
             size="large"
+            value="https://www.youtube.com/watch?v=4F4qzPbcFiA"
```

Comment:
Is this hardcoded value for debug purposes?  Fine with keeping it for now but wanted to ask for the record.

Reply:
yup

Reply:
yup